### PR TITLE
Update images for Spring 2022 based on popularity data

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

**Review by tokkoro — APPROVED** [review]
Imagine all the images

File: images.v2.yaml [hunk 1]
```diff
@@ -1,328 +1,76 @@
...
-    - 'python 3'
-    - 'tensorflow 1.13.1'
-    - 'GPU'
+pytorch/pytorch:1.10.0-cuda11.3-cudnn8-runtime:
+  description: PyTorch 1.10.0 on CUDA 11.3
+  isRecommended: true
+  tags:
+  - PyTorch
+  - PyTorch 1.10.0
+  - GPU
```
Comment (tokkoro) [inline images.v2.yaml:7]: Should we add a CUDA tag? as now there is a small mismatch with description and tags, where one separates between more broadly GPU support and the other is more specifically about CUDA support.

**Review by tokkoro — APPROVED** [review]
Looks good. A small question about CUDA vs. GPU.

File: images.v2.yaml (hunk 1 above)
Comment (akx) [reply images.v2.yaml:7]: We don't really use the tags for anything, so maybe if we do, then...

**Pull request merged**